src/App/Pages/Vault/GroupingsPage/GroupingsPageViewModel.cs

```diff
@@ -76,6 +77,7 @@ public GroupingsPageViewModel()
                 await LoadAsync();
             });
             CipherOptionsCommand = new Command<CipherView>(CipherOptionsAsync);
+            VaultFilterCommand = new Command(async () => await VaultFilterOptionsAsync());
```

Comment:
We should use the `AsyncCommand` from [Xamarin.Community.Toolkit](https://docs.microsoft.com/en-us/xamarin/community-toolkit/objectmodel/asynccommand), it has `onException` as a parameter that's useful to trigger the logger 😄

Reply:
I tried to update the other Commands as well but ended up rolling those back due to some unusual events in the debug log (apparently further adjustments will be needed to make those work cleanly with `AsyncCommand`, skipping that for now since it's not part of this work)